packages/material-ui/src/Badge/Badge.test.js

```diff
@@ -36,361 +37,133 @@ describe('<Badge />', () => {
     }),
   );
 
-  const testChildren = <div className="unique">Hello World</div>;
-
   it('renders children and badgeContent', () => {
-    const wrapper = shallow(<Badge badgeContent={10}>{testChildren}</Badge>);
-
-    assert.strictEqual(wrapper.contains(testChildren), true);
-    assert.strictEqual(wrapper.find('span').length, 2);
+    const children = <div />;
+    const badge = <div />;
+    const wrapper = mount(<Badge badgeContent={badge}>{children}</Badge>);
+    assert.strictEqual(wrapper.contains(children), true);
+    assert.strictEqual(wrapper.contains(badge), true);
```

Comment:
Doesn't this just check that a div is inside the wrapper?

Reply:
The assertion is not correct: https://airbnb.io/enzyme/docs/api/ShallowWrapper/contains.html.

Reply:
Giving the badge an id fixes it

Reply:
Thanks!